cmds/exp/madeye/doc.go

```diff
@@ -0,0 +1,18 @@
+// u-root was intended to be capable of function as a universal root, i.e. a root
+// file system that you could boot from different architectures.
+// We all this Multiple Architecture Device Image, or MADI, pronounced Mad-Eye.
+// Apologies to Harry Potter.
+// This command implements MADI and works as follows:
+// Given a set of images, e.g.
+// initramfs.linux_<arch>.cpio it derives the architecture from the name.
+// It then reads the cpio in.
+// For all symlinks which resolve to absolute paths, it records the file they point to.
+// For a distinguished set of directories, it relocates them from / to /<arch>/, a la Plan 9.
+// If there is a /init, it moves to /<arch>/init.
+// It adjusts absolute path symlinks.
+// future: it looks for conflicting dev entries.
+// It then writes it out.
+// To boot a kernel with a MadEye, one must adjust the init= arg to prepend the architecture.
+// For example, /init on arm would become /arm/init
```

Comment:
does the non-arch init get called still?  (via e.g. arm/init)  not sure if you want that or not.

Reply:
there's no way to call the arm init on the x86 if that's the question?